application/frontend/controllers/SiteController.php

```diff
@@ -29,7 +31,8 @@ public function actionStatus()
             Yii::$app->db->open();
         } catch (Exception $e) {
             \Yii::error($e->getMessage());
-            throw new ServerErrorHttpException(
+            throw new HttpException(
+                self::HttpExceptionBadGateway,
```

Comment:
I'm curious: why a Bad Gateway error if we fail to connect to the database?

Reply:
Good question. I was following a precedent (in idp-pw-api, iirc), but I should have given proper consideration to the meaning of Bad Gateway.

Reply:
503? https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/503